src/core/path.c

```diff
@@ -33,6 +33,11 @@ QuicPathInitialize(
     Path->RttVariance = Path->SmoothedRtt / 2;
     Path->EcnValidationState =
         Connection->Settings.EcnEnabled ? ECN_VALIDATION_TESTING : ECN_VALIDATION_FAILED;
+#ifdef QUIC_USE_RAW_DATAPATH
+    if (MsQuicLib.ExecutionConfig->Flags & QUIC_EXECUTION_CONFIG_FLAG_QTIP) {
+        CxPlatRandom(sizeof(Path->Route.TcpState.SequenceNumber), &Path->Route.TcpState.SequenceNumber);
+    }
+#endif
```

Comment:
Can we offload this to some datapath layer instead?

Reply:
Wrap it in a function called something like CxPlatInitiateDatapathRouteInfo? This is setting the initial sequence number. It has to be set when we initialize the route object.